test/procore/services/batch_sync_test.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class Procore::Services::BatchSyncTest < Minitest::Test
+  def test_execute
+    updates = (0..1000).to_a.map { |count| { id: count, origin_id: "origin_id_#{count}" } }
+    oauth_connection = Minitest::Mock.new
+
+    url = '/erp_projects/sync'
```

Comment:
Even though it's just an example, let's find a public version of the sync endpoint to use here.